src/controllers/auth.ts

```diff
@@ -1,36 +1,107 @@
 import * as express from 'express'
 import {
-  Request,
-  Route,
+  Get,
+  Inject,
   Post,
+  Route,
 } from 'tsoa'
 import {
-  Message,
-  User,
+  UserChange,
+  UserEmail,
   UserLogin,
   UserSignin,
-} from '../models/user'
-import login from '../services/login'
-import signin from '../services/signin'
+  UserToken,
+} from '../models/auth'
+import {
+  change,
+  check,
+  login,
+  logout,
+  me,
+  resend,
+  reset,
+  signin,
+  verify,
+} from '../services/auth'
 
-@Route('auth')
+@Route('user')
```

Comment:
Yep, I hate, but here decorators allows us to generate openAPI specification straight from controllers (via `tsoa`), so I treat this as a lesser evil.